app/controllers/index_controller.rb

```diff
@@ -1,4 +1,7 @@
+require 'news-api'
 class IndexController < ApplicationController
   def index
+    api_key = News.new(Rails.application.credentials.news[:news_api_key])
```

Comment:
The variable naming here is a bit off...

You're initializing the News API client here so variable names like `news_api_client`, `news_client` are probably better suited than `api_key`. You're not storing the API key in that variable